frontend/src/static/css/panels.scss

```diff
@@ -0,0 +1,98 @@
+@import 'colors';
+@import 'z-indices';
+
+.toolbar {
+  cursor: pointer;
+  float: right;
+}
+
+.sorting {
+  align-items: center;
+  display: flex;
+  margin-top: 1.0rem;
+
+  .mui-select {
+    cursor: pointer;
+    padding-right: 15px;
+  }
+
+  label {
+    overflow-y: hidden;
+  }
+
+  & > label {
+    margin-left: 10px;
+  }
+
+  span {
+    margin-left: .25rem;
+  }
+}
+
+.panel-padding {
+  padding: 2rem 1.5rem 2rem 2.2rem
+}
+
+#app-wrapper {
+  display: flex;
+  height: 100vh;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: z-index('app-wrapper');
+
+  #tabs-wrapper {
+    $tabs-width: 50%;
+
+    background: mui-color('white');
+    flex: 0 0 $tabs-width;
+    overflow-y: scroll;
+    position: relative;
+
+    .tab-close {
+      cursor: pointer;
+      position: fixed;
+      top: calc(50vh - 3rem);
+      z-index: z-index('center-divider');
+
+      svg {
+        background-color: mui-color('grey', '300');
+        color: mui-color('black');
+        padding: 3rem .25rem;
+      }
+    }
+
+    .tab-content {
+      display: flex;
+
+      .tab-panes {
+        text-align: left;
+        width: 100%;
+      }
+    }
+  }
+
+  #tab-resize {
```

Comment:
isn't tab-resize within `tabs-wrapper`?

Reply:
Ah my bad. Have updated it. Thanks for noticing it.